tap_hubspot/__init__.py

```diff
@@ -159,17 +162,23 @@ def get_field_schema(field_type, extras=False):
             }
         }
 
-def parse_custom_schema(entity_name, data):
+def parse_custom_schema(entity_name, data, force_extras=None):
+    if force_extras is not None:
+        extras = force_extras
+    else:
+        extras = entity_name != 'contacts'
```

Comment:
This feels weird the way the code is written. Why not make force_extras a boolean instead of an Optional of... I guess a boolean? Have I mentioned I miss type hints in python? I wish we used them... but that's not important right now. I see... you allow force extras to be passed as either true or false. 

Is `entity_name != 'contacts'` really the most common default use case? I would rather you just didn't have a default value and passed it in, you only us this method twice, ever right? So what's the benefit of pushing the logic into the method and not from the caller?

Reply:
> Is entity_name != 'contacts' really the most common default use case?

Yes, it's what it was before. I didn't spend too much thing clarifying the logic, but I needed to the ability to override that value

Reply:
Ok, but why do you need to put this logic inside the method. I see this method called twice below. Why not just pass in `extras = entity_name != 'contacts'` as the second required parameter to this method for the only time it's not set to `extras=False` in the method `get_v3_schema` below?

Reply:
I noticed that the one time I use `force_extras`, the boolean would come out to `false` anyway. So I took out this addition